src/test/java/com/rakovets/course/java/core/practice/strings/StringUtilTest.java

```diff
@@ -0,0 +1,335 @@
+package com.rakovets.course.java.core.practice.strings;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class StringUtilTest {
+    StringUtil stringUtil = new StringUtil();
+
+    //Task 1 tests
+    @Test
+    void concatTwoStringsTest() {
+        String expected = "789123";
+        String actual = stringUtil.concatTwoStrings("789", "123");
+        Assertions.assertEquals(actual, expected);
```

Comment:
Use approach GIVEN empty line WHEN empty line THAN
```suggestion
        String expected = "789123";
        
        String actual = stringUtil.concatTwoStrings("789", "123");
        
        Assertions.assertEquals(actual, expected);
```

Reply:
Fixed, please review